### ISSUE TITLE
DPL-985 Create a new view in the reporting database for sample tracking

### ISSUE THREAD

**Issue by javidahmed64592** [sanger/General-Backlog-Items#351]
**User story**
A view can be created to generate the sample tracking report using the view created in #350. The end result should be a view which can be used in the following way:

```
SELECT * FROM sample_tracking_view
WHERE study_name = 'Study name';
```

The returned result should be the same format as the mock report created in #342 (tab 'Mock report').

**Who are the primary contacts for this story**
TW KT AY JA

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] Identify studies from the past X months
- [x] Create a new view `sample_tracking_view` in the reporting database:
  - [x] `mlwhd_reporting` database in UAT
  - [x] `mlwh_reporting` database in production
- [x] Document this new view in Confluence

**Dependencies**
This story is blocked by the following dependencies:
- #350

References
This story has a non-blocking relationship with:
- #349

**Additional context**
Information about the report can be found in [Sample Tracking Report](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Sample+Tracking+Report).

This story is a part of the roadmap in [Sample Tracking Proof Of Concept](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Sample+Tracking+Proof+Of+Concept).

**Comment by javidahmed64592** [comment]
Now that the views have been optimised, the report is able to be generated in ~30 seconds for one study. As such, I will proceed with these views and add them to the production database.

I've added a page in Confluence outlining all the work I've done on this report and what future work can be done:

[Sample Tracking Report](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Sample+Tracking+Report)
